src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/SasTokenGeneratorServiceTest.java

```diff
@@ -52,7 +52,7 @@ public void should_generate_sas_token_when_service_configuration_is_available()
 
         assertThat(queryParams.get("sig")).isNotNull();//this is a generated hash of the resource string
         assertThat(queryParams.get("se")[0]).startsWith(currentDate);//the expiry date/time for the signature
-        assertThat(queryParams.get("sv")).contains("2018-11-09");//azure api version is latest
+        assertThat(queryParams.get("sv")).contains("2019-02-02");//azure api version is latest
```

Comment:
There are overall 3 places like that. IIRC

Reply:
dammit